meta-lmp-bsp/conf/machine/imx8qm-mek.conf

```diff
@@ -0,0 +1,8 @@
+#@TYPE: Machine
+#@NAME: i.MX 8QM MEK
+#@DESCRIPTION: i.MX 8QuadMax MEK board
+#@MAINTAINER: Igor Opaniuk <igor.opaniuk@foundries.io>
+
+require conf/machine/imx8qm-mek.conf
+
+MACHINEOVERRIDES =. ":imx8qm-mek"
```

Comment:
Why are we adding this board conf in meta-lmp? Isn't the one from meta-freescale enough already?

`meta-freescale/conf/machine/imx8qm-mek.conf`

Reply:
well, that was done for other boards also
for example:
```
xdev@igor-thinkpad:~/foundries.io/reps/lmp.repo/layers/meta-lmp/meta-lmp-bsp/conf/machine$ cat imx8mqevk.conf 
#@TYPE: Machine
#@NAME: NXP i.MX 8M Quad EVK
#@SOC: i.MX8MQ
#@DESCRIPTION: Machine configuration for NXP i.MX 8M Quad Evaluation Kit
#              Done as a transition to the new machine names available in hardknott.
#@MAINTAINER: Ricardo Salveti <ricardo@foundries.io>

require conf/machine/imx8mq-evk.conf

MACHINEOVERRIDES .= ":imx8mq-evk"

```

Reply:
Oh, that was because the machine was renamed. The conf I used is imx8mqevk.conf which includes imx8mq-evk.conf
 (that is so we can keep our factories building without having to rename the machine name).

Here we don't need that because both files are named the same way.

Reply:
Ok, got it. Thanks